packages/widgets/src/blocks/legacy-widget/edit/widget-type-selector.js

```diff
@@ -29,7 +29,7 @@ export default function WidgetTypeSelector( { selectedId, onSelect } ) {
 		<SelectControl
 			__next40pxDefaultSize
 			__nextHasNoMarginBottom
-			label={ __( 'Select a legacy widget to display:' ) }
+			label={ __( 'Legacy widget to display' ) }
```

Comment:
To test, install and enable a legacy theme like Twenty Twelve. Go to `wp-admin/widgets.php`, and insert a Legacy Widget block into one of the sections.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/8cc768cb-26a9-466c-b125-4fb9c7e529f9" alt="Legacy Widget placeholder, before" width="685">|<img src="https://github.com/user-attachments/assets/35a7d72b-9116-4895-895b-60806ef3a069" alt="Legacy Widget placeholder, after" width="685">|

No strong opinion about the label text — I think it could be as short as "Legacy widget".

Reply:
The shorter one makes sense to me; however, note that if we change the string too much, GlotPress will drop existing translations and will no longer consider them "fuzzy." By default, if half the string is the same, its translations will be kept and marked as fuzzy:

https://github.com/GlotPress/GlotPress/blob/7c700cc9e76324460387259b01e38969ce1f074c/gp-includes/things/original.php#L450

With that in mind, what you changed should help keep the string. If you change it to "Legacy widget", the translations will be dropped and the string will have to be re-translated. This might be what you essentially expect, but I just wanted to confirm.

Reply:
I think "Legacy widget" is much better. Could we maybe ping someone in the translation community to take care of it after merged?

Reply:
Translations are a shared responsibility, thousands of folks translate into all languages, and the string will need to just be re-translated in all languages. Not a huge deal, it's happened before. Just feels like something that's not part of the PR's goal and will cause an unintended side effect (losing the existing translations).

Reply:
Interesting, I wasn't thinking about translations! I'm not super familiar with the i18n system, but since there already is a "Legacy widget" string within the [same project](https://translate.wordpress.org/projects/wp/dev/ja/default/?filters%5Bterm%5D=Legacy&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc), I would assume that it would just reuse that? @tyxla Do you happen to know if there are any quirks here?

Reply:
Ah, didn't realize we're using that string already. It appears like it should work! It would only be an issue if the meaning of the string in the two separate contexts is not the same. Looks like it should be OK though 👍

Reply:
Perfect, let's go with the shorter one then.